### ISSUE TITLE
Tests fail with SQLite 3.26+

### ISSUE THREAD

**Issue by brianmay** [Karaage-Cluster/karaage#603]
See https://code.djangoproject.com/ticket/29182

Hopefully this will get fixed with Django 2.2.

Test fails with error like (note table name may vary):

```
django.db.utils.OperationalError: Problem installing fixtures: no such table: institute__old
```

**Comment by brianmay** [comment]
Mysql and postgresql support are fine.

**Comment by brianmay** [comment]
This is fixed now we have moved to Django 2.2
